studio/components/interfaces/Organization/NewOrg/NewOrgForm.tsx

```diff
@@ -296,7 +296,7 @@ const NewOrgForm = ({ onPaymentMethodReset }: NewOrgFormProps) => {
                 <div className="flex items-center space-x-8">
                   <img
                     alt="Card"
-                    src={`${BASE_PATH}/img/payment-methods/${paymentMethod.card.brand
+                    src={`${BASE_PATH}/img/payment-methods/${(paymentMethod.card?.brand || 'visa')
```

Comment:
nit: instead of defaulting to visa, maybe have a placeholder div instead which dimensions matches the image?

e.g paymentMethod.card !== undefined ? <div> : <img... />